### PR TITLE
Added ofxOld.h for deprecated symbols

### PR DESCRIPTION
**Pull request by PierreJasmin**
Did that work?

### CODE REVIEW

File: include/ofxOld.h [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+#define kOfxImageComponentYUVA "OfxImageComponentYUVA"
+
+/** @brief Indicates whether an effect is performing an analysis pass.
+--ofxImageEffects.h
+   - Type - int X 1
+   - Property Set -  plugin instance (read/write)
+   - Default - to 0
+   - Valid Values - This must be one of 0 or 1
+
+This feature has been deprecated - officially commented out v1.4.
```
Comment (garyo) [inline include/ofxOld.h]: Could you add the @deprecated doxygen tag to these?  See my version at https://github.com/garyo/openfx/commit/8e715b2834570f9a8089f032cb8705694a9e0661?w=1 for reference.

Comment (PierreJasmin) [inline include/ofxOld.h]: Do we still need #ifdef DOXYGEN  ?

Comment (garyo) [inline include/ofxOld.h]: No.  That was only when trying to keep the #defines inline -- that's what prompted me to ask for alternative ideas.  I think all you need is the @deprecated. (Plus switch the 1.3 to 1.4, but that should be a separate commit and PR, I can do that if desired.)

Comment (PierreJasmin) [inline include/ofxOld.h]: Added @deprecated -  and leaving until wed., will only have email late 
at night

**Comment by PierreJasmin** [comment]
Great

Will look at it this week-end
I pre-wrote an announcement, will send for review

Etc

-- Confused how this gets updated? -- 
http://openeffects.org/documentation/api_doc/
OFX API v. 1.3 Reference 
<http://openeffects.org/documentation/api_doc>   -- Useful links on 
front page

Pierre

**Pull request merged**